### DISCUSSION TITLE
Forecast not work

### DISCUSSION THREAD

**Discussion by ManuelGlasl** [cytech/Home-Assistant-wundergroundpws#217] [General]
Hi, since my last HA Update my Weather Forecast dont work. 
You can see my Screenshot. 

![Screenshot_20231208-211805](https://github.com/cytech/Home-Assistant-wundergroundpws/assets/125298241/a5ec7775-be67-4aef-93f2-3bf9dd3e6fd2)
![Screenshot_20231208-211814](https://github.com/cytech/Home-Assistant-wundergroundpws/assets/125298241/776c9731-f79c-4176-b31d-7c518bda00ed)

**Comment by rudizl** [comment]
Are you on the last version of the integration? It must be 2.0.4.

**Comment by ManuelGlasl** [comment]
Where can i see it? In Hacs i couldnt find it...

**Comment by rudizl** [comment]
You can add it in Hacs this way:
![Screenshot_20231208_223655_Home Assistant](https://github.com/cytech/Home-Assistant-wundergroundpws/assets/5582186/b02c6f1c-62a4-43f8-b849-f9a112e7ee06)
![Screenshot_20231208_224246_Home Assistant](https://github.com/cytech/Home-Assistant-wundergroundpws/assets/5582186/cd074472-1fc0-4c9f-8346-b66327d843ba)


I know that cytech don't like hacs for sone reason, but even that way I receive wps integration updates without any problems😉

**Comment by ManuelGlasl** [comment]
Ok, thanks! But I have the integration in hacs, Look in the Screenshot 
![Screenshot_20231208-221325](https://github.com/cytech/Home-Assistant-wundergroundpws/assets/125298241/1e543910-0ab1-47d2-b5f9-0682c4885bee)

But i dont see the version... 


https://github.com/cytech/Home-Assistant-wundergroundpws#installation

**Comment by rudizl** [comment]
![Screenshot_20231208_231939_Home Assistant](https://github.com/cytech/Home-Assistant-wundergroundpws/assets/5582186/75a906e2-2ec5-408c-becf-9f146c25abd8)
![Screenshot_20231208_231949_Home Assistant](https://github.com/cytech/Home-Assistant-wundergroundpws/assets/5582186/347973ce-35f8-4264-9174-33dc0a5b2153)

**Comment by ManuelGlasl** [comment]
Thx. Can i download the version 2.04 like the github link or i had to to with the zip file?
